sourced/ml/__main__.py

```diff
@@ -70,6 +90,30 @@ def get_parser() -> argparse.ArgumentParser:
     # Create and construct subparsers
 
     subparsers = parser.add_subparsers(help="Commands", dest="command")
+
+    repo2coocc_parser = subparsers.add_parser(
+        "repo2coocc", help="Produce the co-occurrence matrix from a Git repository.",
+        formatter_class=ArgumentDefaultsHelpFormatterNoNone,
+        parents=[])
+    add_engine_args(repo2coocc_parser)
```

Comment:
Let's be consistent. If we switch from `parents` to functions then we should refactor all the previous `parents`